src/main/java/org/dependencytrack/notification/publisher/SendMailPublisher.java

```diff
@@ -136,7 +140,7 @@ private void sendNotification(final PublishContext ctx, Notification notificatio
             final SendMail sendMail = new SendMail()
                     .from(smtpFrom)
                     .to(destinations)
-                    .subject("[Dependency-Track] " + notification.getTitle())
+                    .subject(smtpPrefix + " " + notification.getTitle() )
```

Comment:
Might be nitpicking, but I suggest to change the name of the variable and setting to something more generic like `emailSubjectPrefix` or something like that. The prefix is not really tied to the SMTP protocol. If other email delivery methods are added in the future, the prefix might still be valid to use, but should not be called `smtpPrefix`.